### PR TITLE
fix(a11y) heading level a11y [TDX-1618]

### PR DESCRIPTION
**Pull request by Mierenga**
### Description

Update h4 tags to h1, while ensuring that they are actually a top-level header within their sections.

### Motivation and Context

This improves the accessibility of the document, so that screen readers can easily determine how a heading relates to a section on the page

### How Has This Been Tested?

This is tested with the axe browser extension

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [ ] are not breaking changes.

### Documentation
- [ ] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/style/_variables.scss [hunk 1]
```diff
@@ -125,7 +125,7 @@ $opblock-box-shadow-color: $black !default;
 $opblock-tag-border-bottom-color: $bright-gray !default;
 $opblock-tag-background-color-hover: $black !default;
 
-$opblock-tab-header-tab-item-active-h4-span-after-background-color: $gray-400 !default;
+$opblock-tab-header-tab-item-active-h1-span-after-background-color: $gray-400 !default;
```
Comment (davidma415) [inline src/style/_variables.scss:128]: Lol these variable names are so long 😂

**Review by davidma415 — APPROVED** [review]
